ingest/lib/index-rectifier.js

```diff
@@ -0,0 +1,186 @@
+// @ts-check
+const pMap = require('p-map');
+
+const dspaceClient = require('../../lib/dspace-client');
+const osClient = require('../../lib/open-search-client');
+const utils = require('./ingest-queue');
+
+module.exports = {
+  /**
+   * Commits index items that have been marked as out of date by
+   * queuing them for re-ingest. Queuing will be done in parallel.
+   *
+   * @param {Object[]} items List of existing index items that need to
+   *                         be updated.
+   * @param {string} ingestTopicArn SNS Topic ARN where new documents
+   *                                are queued.
+   * @param {Object} [options={}] Additional options.
+   * @param {string} [options.region=us-east-1] AWS region containing the
+   *                                            infrastructure.
```

Comment:
What does this return?

Reply:
As written it returns nothing. I can add something if you think we should.